chain/arweave/src/data_source.rs

```diff
@@ -47,6 +49,20 @@ impl blockchain::DataSource<Chain> for DataSource {
         self.source.start_block
     }
 
+    fn handler_kinds(&self) -> HashSet<&str> {
+        let mut kinds = HashSet::new();
```

Comment:
Wondering if this should be a `HashSet` or just a `Vec`. But in `ethereum` we have blockHandlers with filters so its needed to iterate over it and add it to a HashSet to avoid duplication.

At the end way up in the call path, we convert this to a HashSet anyway.